sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -2,39 +2,23 @@
 
```

Comment:
The convention is to call such a file base. Py

Reply:
Done, renamed from `kernel_ridge.py` to `base_ridge.py` 888c59e